geopandas/tests/test_pandas_methods.py

```diff
@@ -279,7 +279,7 @@ def test_numerical_operations(s, df):
     with pytest.raises(TypeError):
         s.max()
 
-    with pytest.raises(TypeError):
+    with pytest.raises((TypeError, ValueError)):
```

Comment:
@martinfleis it might be a good idea to add a comment for crosslink to https://github.com/pandas-dev/pandas/issues/32749

as source of `ValueError` in case that is reverted back to only returning `TypeError`.

Also, it might be a good idea to add a comment noting that `idxmax()` is not supported for geometry column, unless you feel that is sufficiently covered by comment on line 275.

Reply:
Thanks! Added link to pandas and clarified comment.